strawberry_django/mutations/resolvers.py

```diff
@@ -216,19 +216,116 @@ def create(
+
+    for name, value in data.items():
+        field = fields.get(name)
+        create_populate_field = True
+
+        if field is None or value is UNSET:
+            # Dont use these, fallback to model defaults.
+            create_populate_field = False
+            continue
+
+        if isinstance(field, models.FileField):
+            if value is None:
+                # We want to reset the file field value when None was passed in the
+                # input, but `FileField.save_form_data` ignores None values. In that
+                # case we manually pass False which clears the file.
+                value = False  # noqa: PLW2901
+
+            # set filefields at the same time so their hooks can use other set values
+            files.append((field, value))
+            create_populate_field = False
+            continue
+
+        if isinstance(field, (ManyToManyField, ForeignObjectRel)):
+            # m2m will be processed later
+            m2m.append((field, value))
+            create_populate_field = False
+            continue
+
+        if isinstance(field, models.ForeignKey) and isinstance(
+            value,
+            # We are using str here because strawberry.ID can't be used for isinstance
+            (ParsedObject, str),
+        ):
+            value, value_data = _parse_data(  # noqa: PLW2901
+                info,
+                field.related_model,
+                value,
+            )
+            if value is None and not value_data:
+                value = None  # noqa: PLW2901
+            elif value is None:
+                value = field.related_model._default_manager.create(  # noqa: PLW2901
+                    **value_data,
+                )
+            else:
+                update(info, value, value_data, full_clean=full_clean)
+
+        if create_populate_field:
+            create_kwargs.update({name: value})
+
+        # Populate the dummy_instance field for validation purposes.
+        update_field(info, dummy_instance, field, value)
+
+    # Creating the instance directly via .create withouth full-clean will
+    # raise ugly error messages. To make them more user-friendly we want to
+    # have full-clean trigger form-validation style error messages.
+    full_clean_options = full_clean if isinstance(full_clean, dict) else {}
+    if full_clean:
+        dummy_instance.full_clean(**full_clean_options)  # type: ignore
+
+    # Create the instance using the manager create method to respect
+    # manager create overrides for proxy-models.
+    instance = model._default_manager.create(**create_kwargs)
+
+    # Now that the instance has been created, go and assign
+    # files and many2many fields.
+    for file_field, value in files:
+        file_field.save_form_data(instance, value)
+
+    for field, value in m2m:
+        update_m2m(info, instance, field, value)
+
+    # FIXME: Should you really need to refresh the object from db?
+    instance.refresh_from_db()
```

Comment:
In this case, I don't think so.

It is mostly useful for update to make sure we are reloading the latest version of the object, but for create we should be fine without it

Reply:
ok, removed these lines.